docassemble/ALToolbox/al_income.py

```diff
@@ -0,0 +1,1015 @@
+# Based on https://github.com/GBLS/docassemble-income/blob/master/docassemble/income/income.py
+
+from docassemble.base.util import (
+    DAObject,
+    DAList,
+    DAOrderedDict,
+    PeriodicValue,
+    DAEmpty,
+    Individual,
+    comma_list,
+    log,
+    object_name_convert,
+    value,
+)
+from decimal import Decimal
+import re
+import datetime
+import docassemble.base.functions
+import json
+
+
+def al_flatten(listname, index=1):
+    """
+    Return just the nth item in an 2D list. Intended to use for multiple choice
+    option lists in Docassemble. e.g., al_flatten(al_asset_source_list()) will
+    return ['Savings','Certificate of Deposit'...].
+    """
+    return [item[index] for item in listname]
+
```

Comment:
Is this function actually used? It's so trivial that I'm not sure we should keep it.

I also find the prefixing very distracting for the function names. It's not idomatic Python and it's not something we're doing with other toolbox modules.

Reply:
I was trying to differentiate them from the GBLS module or other packages with broad names. I'm happy with whatever, so just let me know what you'd like.

Reply:
If it's not used in a current demo just remove it. Don't prefix functions, only classes.